cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# this script creates all the users/keys on the external cluster
+# those keys will be injected via the import-external-cluster.sh once this one is done running
+# so you can run import-external-cluster.sh right after this script
+set -Eeuo pipefail
+
+#############
+# VARIABLES #
+#############
+
+: "${CLIENT_CHECKER_NAME:=client.checker}"
```

Comment:
how about a default username of `client.healthchecker`? The name `checker` is too generic to understand.